### DISCUSSION TITLE
I want to be able to revert to an earlier change request

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#126] [Feature Requests]
### What's your feature/idea?

I want possibility in gitbook to revert to a previously committed change request and overwrite the latest changes with whatever change request from history.  

### What's your desired outcome?

Be able to choose a change request from history and make it the latest.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

Avoid manually revert the changes. Gitbook should provide the possibility to revert back to earlier changes if needed.

**Comment by aikoii** [comment]
Hi @shimshim938!

Unless I'm missing something, you should be able to roll back to a previous point in time 🙂.

We have helpful documentation, where you’ll find more info:

» https://docs.gitbook.com/tour/activity-history#rolling-back-to-a-previous-version

If you have trouble with it, please email us at support@gitbook.com 💌

**Comment by ahcsco** [comment]
Hi,
Thanks for info. Yes it looks like the "secret" rollback option exists
after much looking around.
The documentation is not very helpful though. Step by step instructions on
how to do a rollback are missing. There is just a GUI snap which itself is
old.
The option is available under change history.

Please do close this thread.

**Reply by aikoii** [reply]
Agree! Our documentation needs some love. We're actively working on improving our docs to make them more helpful 😊 👍!
